bin/create_metabinner_bins.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+## Originally written by Hesham Almessady (@HeshamAlmessady) and Adrian Fritz (@AlphaSquad) in https://github.com/hzi-bifo/mag and released under the MIT license.
+## See git repository (https://github.com/nf-core/mag) for full license text.
+
+import sys
+import os
+from Bio import SeqIO
+
+binning = sys.argv[1]
+fasta = sys.argv[2]
+path = sys.argv[3]
+prefix = sys.argv[4]
+length = sys.argv[5]
+
+try:
+    os.mkdir(path)
+except OSerror as error:
+    pass
+
+Metabinner_bins = {}
+with open(binning, 'r') as b:
+    for line in b:
+        contig, bin = line.strip().split('\t')
+        Metabinner_bins[contig] = bin
+
+for outf in set(Metabinner_bins.values()):
```

Comment:
I don't think this block is really needed, because opening the files in append mode will create them if they don't exist (`.tooShort.fa` and `.unbinned.fa` are already doing that).